CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
   * raised minimum required PHP version to 7.2.0
   * all methods of `org\bovigo\vfs\visitor\vfsStreamVisitor` are now declared with `self` as return type
   * `vfsStreamWrapper::setRoot()` and `vsfStreamWrapper::getRoot()` method signatures now require and return `org\bovigo\vfs\vfsStreamDirectory` vice `org\bovigo\vfs\vfsStreamContainer`.
+  * `vfsStream::newFile()` and `org\bovigo\vfs\vfsStreamFile` will throw an exception if the filename contains a forward slash (`/`).
```

Comment:
Since `vfsStreamBlock` is marked as `@api`, should this entry also mention this new behavior?

Reply:
Added